static/app/gettingStartedDocs/javascript/javascript.tsx

```diff
@@ -149,6 +354,70 @@ const onboarding: OnboardingConfig = {
       guideLink: 'https://docs.sentry.io/platforms/javascript/sourcemaps/',
     }),
   ],
+  verify: getVerifyConfig,
```

Comment:
at the moment the verify codes for the loader and package manager are different, but we can have the same for both.